AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -48,18 +48,54 @@ struct SessionHeaderView: View {
 
 private extension SessionHeaderView {
     var dateAndTime: some View {
-        guard let start = session.startTime else {
-            return Text("")
-        }
-        let end = session.endTime ?? Date()
         
         let formatter = DateIntervalFormatter()
-        
         formatter.timeStyle = .short
         formatter.dateStyle = .medium
-        let string = DateIntervalFormatter().string(from: start, to: end)
-        let replaced = string.replacingOccurrences(of: "—", with: "-")
-        return Text(replaced)
+        var date = ""
+        var additionalDate = ""
+        
+            guard let start = session.startTime else { return Text("") }
+            let end = session.endTime ?? Date()
+            let string = DateIntervalFormatter().string(from: start, to: end)
+        
+        if is24Hour() {
+            let replaced = string.replacingOccurrences(of: "—", with: "-")
```

Comment:
I'm not too sure this line is doing anything? It's replacing occurrences of "-" with "-" ?

Reply:
The longer "_" with the shorter "-"